cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -41,7 +41,6 @@ import (
 
 	"k8s.io/api/core/v1"
 	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/kubernetes"
```

Comment:
@caesarxuchao do you have a thought for the canonical import name?  `kubernetes`?  `clientset`?  something else?

Reply:
When it's imported outside of kubernetes, it makes sense to call the package `kubernetes`, because it might be used together with other generated clientsets, e.g.  apiextension.clientset.

When used inside kubernetes project, calling it "clientset" is the better.

Reply:
> When it's imported outside of kubernetes, it makes sense to call the package kubernetes, because it might be used together with other generated clientsets, e.g. apiextension.clientset.
> 
> When used inside kubernetes project, calling it "clientset" is the better.

Sounds reasonable to me.